src/Microsoft.Identity.Web/TokenCacheProviders/Distributed/MsalDistributedTokenCacheAdapter.cs

```diff
@@ -82,6 +82,12 @@ protected override async Task RemoveKeyAsync(string cacheKey)
             catch (Exception ex)
             {
                 _logger.LogError($"[IdWebCache] Connection issue encountered with Distributed cache. Currently using In Memory cache only. Error message: {ex.Message} ");
```

Comment:
>Currently using In Memory cache only [](start = 101, length = 36)

Forgot to mention this last time...
Replace: Remove cache entry for (key) failed. Don't have to mention anything about the memory cache here.
(same comment a few other places)

Reply:
done & i agree...was more for testing too to see where we were. 

---
In reply to: [587807465](https://github.com/AzureAD/microsoft-identity-web/pull/1046#discussion_r587807465) [](ancestors = 587807465)